### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by Jawnnypoo**
Latest build dependencies

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -14,3 +14,5 @@ POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=instacart
 POM_DEVELOPER_NAME=Instacart
+
+android.useAndroidX=true
```
Comment (Jawnnypoo) [inline gradle.properties:18]: This is required now

File: formula-android/build.gradle [hunk 2]
```diff
@@ -5,7 +5,6 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: "com.github.ben-manes.versions"
 apply plugin: "com.vanniktech.maven.publish"
-apply plugin: 'org.jetbrains.dokka-android'
```
Comment (Jawnnypoo) [inline formula-android/build.gradle:8]: Errors were appearing since `com.vanniktech.maven.publish` includes this plugin already.

**Comment by carrotkite** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Total coverage of 55.46%. Improve this to at least 69%</td>
    </tr>
  </tbody>
</table>

### JaCoCO Code Coverage 55.46% :warning:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: build.gradle [hunk 3]
```diff
@@ -21,14 +21,14 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "com.github.laimiux:gradle-android-junit-jacoco-plugin:a910712557"
+        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
```
Comment (Laimiux) [inline build.gradle:26]: Maybe this caused the issue with jacoco coverage?

Comment (Jawnnypoo) [reply build.gradle:26]: There is a different error we get if we update the other plugins and don't update this one too:

Screen Shot 2020-06-25 at 3 32 58 PM

File: build.gradle [hunk 4]
```diff
@@ -45,8 +45,23 @@ task clean(type: Delete) {
...
+            '**/*$inlined$*.*', // Kotlin specific, Jacoco can not handle several "$" in class name.
+            '**/com/instacart/formula/test/**/*.*'
+    ]
+    ignoreProjects = [
+            "samples/*"
+    ]
     includeNoLocationClasses = true
+    includeInstrumentationCoverageInMergedReport = true
 }
+apply from: 'gradle/merge-reports.gradle'
```
Comment (Jawnnypoo) [inline build.gradle:67]: I **think** this reordering is actually what fixed this.

**Pull request merged**